ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -575,28 +575,46 @@ def _serialize_workflow_trigger(self) -> Optional[JsonArray]:
             trigger_id = trigger_class.__id__
 
             # Serialize trigger attributes like node outputs
-            attribute_references = trigger_class.attribute_references().values()
-            trigger_attributes: JsonArray = cast(
-                JsonArray,
-                [
-                    cast(
-                        JsonObject,
-                        {
-                            "id": str(reference.id),
-                            "name": reference.name,
-                            "type": primitive_type_to_vellum_variable_type(reference),
-                            "value": None,
-                        },
-                    )
-                    for reference in sorted(attribute_references, key=lambda ref: ref.name)
-                ],
-            )
+            trigger_data: JsonObject
+            if trigger_type == WorkflowTriggerType.SCHEDULED:
+                # For scheduled triggers, attributes should be empty
+                # and cron/timezone should be top level
+
+                config_class = trigger_class.Config
+                cron_value = getattr(config_class, "cron", None)
+                timezone_value = getattr(config_class, "timezone", None)
+
+                trigger_data = {
+                    "id": str(trigger_id),
+                    "type": trigger_type.value,
+                    "cron": cron_value,
+                    "timezone": timezone_value,
+                    "attributes": [],
+                }
+            else:
+                # For other triggers, serialize attributes from attribute_references
+                attribute_references = trigger_class.attribute_references().values()
```

Comment:
[oos] would love to support `__iter__` on trigger_class then reserving a special name `attribute_references`